fiftyone/core/labels.py

```diff
@@ -129,18 +201,10 @@ class Classification(ImageLabel):
 
     meta = {"allow_inheritance": True}
 
-    _id = fof.ObjectIdField(
-        required=True, default=ObjectId, unique=True, primary_key=True
-    )
```

Comment:
I'm able to reliably reproduce a failure in `SerializationTests.test_embedded_document` locally (with `pytest tests/unittests.py -k test_embedded_document`).
I'm guessing the issue is that you've removed the `_id` field here but haven't made `Classification` inherit from `_HasID`. Not sure what else might need to change here.

Reply:
Thanks for catching this! I fixed it